docs/developer-docs/integrations/sns/integrate-sns/index-integration.md

```diff
@@ -1 +1,51 @@
-Title h1: Considerations to Integrate with Index Canister
\ No newline at end of file
+# Index Canister
+
+The index canister keeps track of each transaction stored by the 
```

Comment:
should we maybe call it the "SNS index canister" and / or write one sentence that this is a canister that is deployed with each SNS and why this is the case? Something like:
- Each SNS is deployed with an index canister that keeps track of each transaction stored by the ledger canister, sorted by ledger accounts. 
- Many wallets / FE integration will want to filter transactions by user / account. as ledger does not keep such a map would have to cache this locally. To facilitate easier FEs, we thus have this cansiter..

Reply:
Added a line in that sense.